### PR TITLE
719 implement the SQL connector

### PR DESCRIPTION
**Pull request by johnBgood**
## Description

SQL Connector initial implementation using the [Jdbi](https://jdbi.org/releases/3.45.1/#_introduction_to_jdbi_3) client.
This connector must allow users to execute CRUD SQL statements with Postgre, MySQL, or Microsoft SQL Server.
Additionally, we can accept "CREATE" statements (for tables, or databases).

## Related issues

Parent ticket: https://github.com/camunda/team-connectors/issues/716
closes https://github.com/camunda/team-connectors/issues/719

### CODE REVIEW

File: connectors/jdbc/src/main/java/io/camunda/connector/jdbc/model/request/connection/DetailedConnection.java [hunk 1]
```diff
@@ -9,24 +9,37 @@
...
     @NotBlank @TemplateProperty(group = "connection", label = "Port") String port,
     @TemplateProperty(group = "connection", label = "Username") String username,
-    @TemplateProperty(group = "connection", label = "Password") String password)
+    @TemplateProperty(group = "connection", label = "Password") String password,
+    @TemplateProperty(group = "connection", label = "Database name") String databaseName,
+    @TemplateProperty(
+            group = "connection",
+            label = "Properties",
+            description =
+                "Provide the payload for the event as JSON. Details in the <a href=\"https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/amazon-eventbridge/?awseventbridge=outbound\" target=\"_blank\">documentation</a>")
```
Comment (johnBgood) [inline connectors/jdbc/src/main/java/io/camunda/connector/jdbc/model/request/connection/DetailedConnection.java:28]: This link will be changed when the docs will be updated

File: connectors/jdbc/src/main/java/io/camunda/connector/jdbc/model/request/connection/DetailedConnection.java [hunk 2]
```diff
@@ -9,24 +9,37 @@
...
     @NotBlank @TemplateProperty(group = "connection", label = "Port") String port,
     @TemplateProperty(group = "connection", label = "Username") String username,
-    @TemplateProperty(group = "connection", label = "Password") String password)
+    @TemplateProperty(group = "connection", label = "Password") String password,
+    @TemplateProperty(group = "connection", label = "Database name") String databaseName,
+    @TemplateProperty(
+            group = "connection",
+            label = "Properties",
+            description =
+                "Provide the payload for the event as JSON. Details in the <a href=\"https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/amazon-eventbridge/?awseventbridge=outbound\" target=\"_blank\">documentation</a>") // TODO description
```
Comment (igpetrov) [inline connectors/jdbc/src/main/java/io/camunda/connector/jdbc/model/request/connection/DetailedConnection.java:28]: I believe, the link to doc is incorrect

**Review by igpetrov — CHANGES_REQUESTED** [review]
Looks good already. Happy to give an approval once all `TODO`s and commented code resolved.

File: connectors/jdbc/src/main/java/io/camunda/connector/jdbc/model/request/connection/DetailedConnection.java (hunk 2 above)
Comment (johnBgood) [reply connectors/jdbc/src/main/java/io/camunda/connector/jdbc/model/request/connection/DetailedConnection.java:28]: indeed, there will a documentation PR and then I'll update the code accordingly :)

File: connectors/jdbc/element-templates/hybrid/jdbc-outbound-connector-hybrid.json [hunk 3]
```diff
@@ -178,27 +194,51 @@
...
+    "condition" : {
+      "property" : "connection.authType",
+      "equals" : "detailed",
+      "type" : "simple"
+    },
+    "type" : "String"
+  }, {
+    "id" : "connection.properties",
+    "label" : "Properties",
+    "description" : "Provide the payload for the event as JSON. Details in the <a href=\"https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/amazon-eventbridge/?awseventbridge=outbound\" target=\"_blank\">documentation</a>",
```
Comment (sbuettner) [inline connectors/jdbc/element-templates/hybrid/jdbc-outbound-connector-hybrid.json:216]: Description seems to be outdated.

Comment (johnBgood) [reply connectors/jdbc/element-templates/hybrid/jdbc-outbound-connector-hybrid.json:216]: That's my bad, I should've mentioned that reviewers shouldn't pay attention to the description for now. I'm writing the documentation, and will add the relevant links where applicable :) 
Same goes for the //TODO

**Review by chillleader — APPROVED** [review]
Nice!

**Pull request merged**